__e2e__/create-new-wallet.e2e.js

```diff
@@ -0,0 +1,70 @@
+import { Application } from 'spectron'
+import path from 'path'
+import test from 'ava'
+
+let electronPath = path.join(__dirname, '..', 'node_modules', '.bin', 'electron')
+if (process.platform === 'win32') {
+  electronPath += '.cmd'
+}
+const appPath = path.join(__dirname, '..')
+
+const app = new Application({
+  path: electronPath,
+  args: [appPath]
+})
+
+test.before(async () => {
+  await app.start()
+  await app.client.waitUntilWindowLoaded()
+})
+
+test.after(async () => {
+  if (app && app.isRunning()) {
+    await app.stop()
+  }
+})
+
+function timeout (ms) {
+  return new Promise(resolve => setTimeout(resolve, ms))
+}
+
+test.serial('should create new wallet', async t => {
+  // Go to create wallet page
+  await app.client.waitForVisible('a[href="/create"]>div.linkBox')
+  await app.client.click('a[href="/create"]>div.linkBox')
+
+  // Enter passphrase twice
+  await app.client.setValue('#newWallet form div:nth-child(1) input', 'test')
+  await app.client.setValue('#newWallet form div:nth-child(2) input', 'test')
+
+  // Click on generate keys button
+  await app.client.click('#newWallet button[type="submit"]')
+
+  // Verify that wallet key list is shown
+  await app.client.waitForVisible('#newWallet .keyList')
+  t.true(await app.client.isVisible('#newWallet .keyList'))
+})
+
+test.serial('should show wallet passphrase', async t => {
+  t.true(await app.client.isVisible('#newWallet .keyListItem:nth-child(1) .key'))
+  const passphraseText = await app.client.getText('#newWallet .keyListItem:nth-child(1) .key');
+  t.truthy(passphraseText.length)
+})
+
+test.serial('should show wallet public address', async t => {
+  t.true(await app.client.isVisible('#newWallet .keyListItem:nth-child(3) .key'))
+  const publicAddressText = await app.client.getText('#newWallet .keyListItem:nth-child(3) .key');
+  t.truthy(publicAddressText.length)
```

Comment:
We could probably leverage `neon-js` to validate that values like public key, address, etc. are being set correctly.  This is still a big improvement over the current lack of test, so I'm going to merge. 😄